llvm/test/Transforms/SimplifyCFG/X86/switch_to_lookup_table.ll

```diff
@@ -2068,3 +2068,25 @@ cond.end:                                         ; preds = %entry, %cond.false
   %conv = sext i3 %cond to i8
   ret i8 %conv
 }
+
+; Don't create a table with an unknown type
+define { i8, i8 } @test_unknown_result_type(i8 %n) {
+; CHECK-LABEL: @test_unknown_result_type(
+; CHECK-NEXT: entry:
+; CHECK-NEXT: switch
```

Comment:
You should use [llvm/utils/update_test_checks.py](https://github.com/llvm/llvm-project/blob/main/llvm/utils/update_test_checks.py), for transform tests especially for which have `; NOTE: Assertions have been autogenerated by utils/update_test_checks.py` for L1.

Reply:
oh I see. Thanks for the tip!